iso15118/secc/states/iso15118_20_states.py

```diff
@@ -1327,6 +1327,26 @@ async def process_message(
                 bpt_dynamic_params = await self.comm_session.evse_controller.get_ac_charge_loop_params_v20(  # noqa
                     ControlMode.DYNAMIC, ServiceV20.AC_BPT
                 )  # noqa
+
+                # For now we just do this for the Dynamic Mode
+                ev_bpt_charge_parameters = ac_charge_loop_req.bpt_dynamic_params.dict()
+                # extract only the power limits and convert them to decimal
+                # representation
+                ev_power_limits = {}
+                for k, v in ev_bpt_charge_parameters.items():
+                    if v and v.get("exponent") is not None:
```

Comment:
Something minor - As the RationalNumber field here is received from the EV side, and as the field is not optional  - so I think the "if v and v.get("exponent")" check is being double safe but not required?

Reply:
you are right, the exponent check is not needed, but the value check it is, cause there are None values:

```
dict_items([('departure_time', None), ('ev_target_energy_request', {'exponent': 3, 'value': 40}), ('ev_max_energy_request', {'exponent': 3, 'value': 60}), ('ev_min_energy_request', {'exponent': 3, 'value': -20}), ('ev_max_charge_power', {'exponent': 3, 'value': 300}), ('ev_max_charge_power_l2', None), ('ev_max_charge_power_l3', None), ('ev_min_charge_power', {'exponent': 0, 'value': -100}), ('ev_min_charge_power_l2', None), ('ev_min_charge_power_l3', None), ('ev_present_active_power', {'exponent': 3, 'value': 200}), ('ev_present_active_power_l2', None), ('ev_present_active_power_l3', None), ('ev_present_reactive_power', {'exponent': 3, 'value': 20}), ('ev_present_reactive_power_l2', None), ('ev_present_reactive_power_l3', None), ('ev_max_discharge_power', {'exponent': 3, 'value': 11}), ('ev_max_discharge_power_l2', None), ('ev_max_discharge_power_l3', None), ('ev_min_discharge_power', {'exponent': -3, 'value': 1}), ('ev_min_discharge_power_l2', None), ('ev_min_discharge_power_l3', None), ('ev_max_v2x_energy_request', None), ('ev_min_v2x_energy_request', None)])
```